CONTRIBUTING.md

```diff
@@ -25,19 +34,19 @@ resource for your first contribution, join our Slack channel and find a buddy!
 
 Your OpenTelemetry buddy is your resource to talk to directly on all aspects of
 contributing to OpenTelemetry: providing context, reviewing PRs, and helping
-those get merged. Buddies will not be available 24/7, but is committed to
-responding during their normal contribution hours.
+those get merged. Buddies will not be available 24/7, but are committed to
+responding during their normal working hours.
 
 ## Development Environment
 
-You can contribute to this project from a Windows, macOS or Linux machine.
+You can contribute to this project from a Windows, macOS, or Linux machine.
 
 On all platforms, the minimum requirements are:
 
-* Git client and command line tools.
-* .NET 8.0+
+* Git client and command line tools
+* [.NET SDK (latest stable version)](https://dotnet.microsoft.com/download)
```

Comment:
is there a need to install latest preview when nearing the .NET release date?

Reply:
I don't think in this repo we should really ever be using a pre-release SDK. In main repo that is more valid for sure.